README.md

```diff
@@ -143,6 +143,14 @@ allow_concatenation = true
 # recommendations without the dashes. This is less common.
 allow_dashed = false
 
+[NlpRules]
+# Allows the user to override the default included
+# exports of LanguageTool, with other custom
```

Comment:
```suggestion
# exports of NlpRules, with other custom
```

Reply:
The rules are actually exports of the LanguageTool :upside_down_face: